scripts/jenkins/ardana/gerrit/gerrit_review.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import sys
+
+sys.path.append(os.path.dirname(__file__))
+from gerrit import GerritChange  # noqa: E402
+from gerrit_project_map import gerrit_project_map  # noqa: E402
+
+
+def gerrit_review(change_no, patch=None, label=None, vote=1, message=''):
```

Comment:
This should probably take in a change object rather than a change_no as most of the places it is called the object is already loaded.